### ISSUE TITLE
Version bump: 1.9.4

### ISSUE THREAD

**Issue by krissen** [sdaps/gentoo-overlay#1]
Thanks for providing this overlay!

I successfully installed 1.9.0, using the ebuild available on this repo. It was also possible to version bump to 1.9.1 by simply copying the ebuild to the relevant filename. However, it seems that subsequent releases are different (also reflected in the substantially smaller filesizes of the tar.gz), and the 1.9.0 ebuild no longer works "as is".

Would be great if latest versions (currently 1.9.4) could be added!

**Comment by benzea** [comment]
Yeah, the python 3 port happened, so that will require some work to get the new dependencies right and such.

I am not sure how active Florian is these days, it may be good if someone else picks it up.
